CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 2.8.0
```

Comment:
I don't have strong feeling about it, but since we are non-backwards-compatible for the GKE stuff, should this be `3.0.0`?

Reply:
sure - we can do that